### ISSUE TITLE
Visualize is broken for scala 2.12.2

### ISSUE THREAD

**Issue by jcranky** [zalando/grafter#63]
The error message: 

```scala
[error] (run-main-0) java.lang.ClassCastException: scala.collection.immutable.Vector cannot be cast to scala.collection.immutable.Queue
java.lang.ClassCastException: scala.collection.immutable.Vector cannot be cast to scala.collection.immutable.Queue
	at scala.collection.immutable.Queue.$plus$plus(Queue.scala:111)
	at org.bitbucket.inkytonik.kiama.relation.Relation$.loop$1(Relation.scala:61)
	at org.bitbucket.inkytonik.kiama.relation.Relation$.fromOneStepGraph(Relation.scala:64)
	at org.bitbucket.inkytonik.kiama.relation.Relation$.fromOneStep(Relation.scala:73)
	at org.zalando.grafter.Visualize.asDotString(Visualize.scala:20)
	at org.zalando.grafter.Visualize.asDotString$(Visualize.scala:16)
	at org.zalando.grafter.Visualize$.asDotString(Visualize.scala:82)
	at org.zalando.grafter.VisualizeSyntax$VisualizeSyntaxOps.asDotString(Visualize.scala:111)
	at org.zalando.grafter.VisualizeSyntax$VisualizeSyntaxOps.asDotString(Visualize.scala:107)
	at com.example.grafterhttp4sexample.main.Visualize$.delayedEndpoint$com$example$grafterhttp4sexample$main$Visualize$1(Visualize.scala:13)
	at com.example.grafterhttp4sexample.main.Visualize$delayedInit$body.apply(Visualize.scala:5)
	at scala.Function0.apply$mcV$sp(Function0.scala:34)
	at scala.Function0.apply$mcV$sp$(Function0.scala:34)
	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
	at scala.App.$anonfun$main$1$adapted(App.scala:76)
	at scala.collection.immutable.List.foreach(List.scala:389)
	at scala.App.main(App.scala:76)
	at scala.App.main$(App.scala:74)
	at com.example.grafterhttp4sexample.main.Visualize$.main(Visualize.scala:5)
	at com.example.grafterhttp4sexample.main.Visualize.main(Visualize.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)

```

To reproduce:

1. create a new application with [my template](https://github.com/jcranky/grafter-http4s.g8), and be sure to set the scala version to _2.12.2_ when prompted
2. run `sbt run`
3. choose `Visualize`

**Comment by etorreborre** [comment]
[Reported to Kiama](https://bitbucket.org/inkytonik/kiama/issues/79/classcastexception-with-scala-2122) but this is [a Scala 2.12.2 bug](https://github.com/scala/bug/issues/10298).

**Comment by Ichoran** [comment]
@etorreborre - Workaround is to make the RHS be a queue also - Kiama could make this change if it needed a fix quickly to support 2.12.2.  I don't feel like signing up for a Bitbucket account, so I can't post on the Kiama issue tracker.

**Comment by etorreborre** [comment]
Tony should release a new Kiama RC with [the fix](https://bitbucket.org/inkytonik/kiama/commits/63a4433) really soon.

**Comment by etorreborre** [comment]
Fixed with #67
